src/System.Private.ServiceModel/src/System/ServiceModel/SpnEndpointIdentity.cs

```diff
@@ -1,37 +1,31 @@
 // Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System.IdentityModel.Claims;
+using System.Runtime;
+using System.Security.Principal; 
+
 namespace System.ServiceModel
 {
     public class SpnEndpointIdentity : EndpointIdentity
-    {
-        private static TimeSpan s_spnLookupTime = TimeSpan.FromMinutes(1);
```

Comment:
Question only: is this code being removed because it is dead?  I was comparing to full framework and was curious.